src/librustc/ty/mod.rs

```diff
@@ -1371,6 +1371,7 @@ pub struct AdtDef {
     pub variants: Vec<VariantDef>,
     flags: AdtFlags,
     pub repr: ReprOptions,
+    pub non_exhaustive: bool,
```

Comment:
👍

Reply:
Can you add a comment briefly explaining what this is, perhaps linking to the RFC? I know none of the other fields have comments, but that's no reason not to do better... =)

Reply:
I definitely will, I plan on doing another pass to make sure everything is well commented, just wanted to get that pushed up before I called it a night. Will do this ASAP.

Reply:
Just pushed up a comment here.